tools/demo/graph.ts

```diff
@@ -134,6 +134,28 @@ class DemoTaskGraph implements TaskGraph {
     on('watchGeolocationSamplesAcquired', run('writeRecords'));
     on('watchGeolocationSamplesAcquired', run('trackEvent'));
     on('watchGeolocationSamplesAcquired', run('stopDetectingWatchGeolocationChanges'));
+
+    on(
+      'startEvent',
+      run('sendPlainMessageToWatch', {
+        message: 'I do not expect a response :)!',
+      }).now()
+    );
+    on(
+      'startEvent',
+      run('sendPlainMessageToWatchAndAwaitResponse', {
+        message: 'PING!',
+      })
+        .every(1, 'minutes')
+        .cancelOn('stopEvent')
+    );
+
+    on('plainMessageSent', run('writeRecords'));
+    on('plainMessageSent', run('trackEvent'));
```

Comment:
I think we already discuss this (not completely sure). I think we decided to not create a record for a sent notification because we were already tracing it, and then we were able to know which notification was sent looking at the tap/discard records. 

In this case, I decided to create a record for this because is the only way to know (in the smartphone side) that the message has been sent, and also to be able to react to that action (e.g., warn the user to look at the watch or something)...

Reply:
Yes, you're right. But now I'm thinking that for that we depend on either of these two things: we force people to review the traces or we rely on the user tapping or discarding a notification. Otherwise, you don't know if the notification was sent or not. This is something to evaluate in the future, for sure.

Unrelated to this PR, what you've done here makes perfectly sense and it's well thought 😉